mapviz_plugins/ui/measuring_config.ui

```diff
@@ -6,38 +6,45 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>198</width>
-    <height>132</height>
+    <width>348</width>
+    <height>128</height>
    </rect>
   </property>
-  <property name="sizePolicy">
-   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
```

Comment:
Main reason of this PR to change `sizePolicy` from Fixed to Preferred, to allow automatic scaling.